src/cli/cli.ts

```diff
@@ -660,6 +660,10 @@ function printHelp(helpMode: string, exitCode: number = 0): void {
     printCommandHelp(helpMode);
   }
   else {
+    if (cli.currentCommandName) {
+      cli.error(`Command '${cli.currentCommandName}' not found. Please find a list of available commands below.`);
+    }
```

Comment:
We'll need to extend this logic a bit more. At the moment, when we execute commands like `m365 spo` or `m365 spo --help`, this error is thrown as well, which seems a bit verbose. Let's look into this a bit more.

Reply:
Makes sense. Added this check